samples/boards/nrf/dmm/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(hello_world)
```

Comment:
I'm not sure if this should be a sample or a test. I don't think anyone would use `dmm` from an application code. It is used only from the drivers' code. So a sample showing how to use it from the application code seems to clutter the sample's folder.